app/src/main/java/com/omkarmoghe/pokemap/views/map/MapWrapperFragment.java

```diff
@@ -199,7 +199,7 @@ private void initMap() {
                 new AlertDialog.Builder(getActivity())
                         .setTitle(getString(R.string.enable_location_permission_title))
                         .setMessage(getString(R.string.enable_location_permission_message))
-                        .setPositiveButton("OK", null)
+                        .setPositiveButton(getString(R.string.button_ok), null)
```

Comment:
wasn't in the code when I did this.  as mentioned in my initial comment, a fair amount of the string references were removed by another PR merge because the requester didn't make sure to update his code-base first.